src/js/entities-service/entities/groups.js

```diff
@@ -11,6 +13,51 @@ const _Model = BaseModel.extend({
   getClinicians() {
     return Radio.request('entities', 'clinicians:collection', this.get('_clinicians'));
   },
+  addClinician(clinician) {
+    const url = `/api/groups/${ this.id }/relationships/clinicians`;
+    const groups = clinician.get('_groups') || [];
+
+    clinician.set({ _groups: groups.concat({ id: this.id }) });
+
+    this.set({ _clinicians: this.get('_clinicians').concat({
```

Comment:
so had a thought here and above.. I _think_ that `_clinicians` isn't going to get  a change event because `concat` modifies the data on the attributes.  So maybe `_.union` would get around that along with matching the `_.reject` pattern below.